fmriprep/cli/run.py

```diff
@@ -109,7 +125,7 @@ def create_workflow(opts):
         'ignore': opts.ignore,
         'skip_native': opts.skip_native,
         'freesurfer': opts.freesurfer,
-        'reportlets_dir': op.join(op.abspath(opts.work_dir), 'reportlets'),
+        'reportlets_dir': op.join(op.abspath(opts.work_dir), 'reportlets')
```

Comment:
Is this an intentional change?

I typically like having the comma there, so that adding a new line only puts one line changed in the diff, not two.